client/src/components/comparison/Comparison.jsx

```diff
@@ -1,9 +1,100 @@
-import React from 'react';
+import React, {useEffect, useState/*, useContext*/} from 'react';
+import RelatedProducts from './RelatedProducts.jsx';
+import UserOutfitItems from './UserOutfitItems.jsx';
+// import {products} from '../../../mock-data/products.js';
+// import {productStyles} from '../../../mock-data/productStyles.js';
+import Requester from '../../Requester.js';
+import LoadingSpinner from '../../components/shared/LoadingSpinner.jsx';
 
-const Comparison = (props) => {
-  return (
-    <div>Comparison</div>
-  );
-};
+const rerequester = Requester();
+
+class Comparison extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      relatedProducts: this.props.relatedProducts,
+      detaledRelatedProductsArr: [],
```

Comment:
'detaled' is a typo, should be 'detailed'. You'll want to make sure you change this wherever it's referenced.